status/reporters.go

```diff
@@ -452,6 +452,14 @@ func (r *GitHubReporter) ReportStatusForSnapshot(k8sClient client.Client, ctx co
 		return fmt.Errorf("sha label not found %q", gitops.PipelineAsCodeSHALabel)
 	}
 	integrationTestStatusDetails := statuses.GetStatuses()
+	latestUpdateTime, _ := gitops.GetLatestUpdateTime(snapshot)
```

Comment:
Maybe we should at least to log err in error log when it fail to fetch timestamp

Reply:
it's actually there, this line just miss err variable
```suggestion
	latestUpdateTime, err := gitops.GetLatestUpdateTime(snapshot)
```